plugin-customcommand/lxqtcustomcommand.cpp

```diff
@@ -95,6 +95,7 @@ void LXQtCustomCommand::settingsChanged()
     QString oldFont = mFont;
     QString oldCommand = mCommand;
     bool oldRunWithBash = mRunWithBash;
+    bool oldOutputImage = mOutputImage;
```

Comment:
There seems to be a problem in the original code, which this patch inherits: except for `mFirstRun` and `mAutoRotate`, no `bool` or `int` variable is initialized in the constructor. This could result in an undefined behavior.

Would you please correct that for all `bool` or `int` variables?